cmd/wallet.go

```diff
@@ -22,8 +29,78 @@ to quickly create a Cobra application.`,
 	// },
 }
 
+// createCmd represents the create command
+var createCmd = &cobra.Command{
+	Use:   "create",
+	Short: "Generate a new wallet file",
+	Long:  `Create a new wallet file containing a single, randomly-generated account.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		w := wallet.NewWallet()
+
+		fmt.Print("Enter a secure password (optional but strongly recommended): ")
+		password, err := password.Read(os.Stdin)
+		fmt.Println()
+		cobra.CheckErr(err)
+		wk := wallet.NewKey(wallet.WithRandomSalt(), wallet.WithPbkdf2Password([]byte(password)))
+		err = os.MkdirAll(common.DotDirectory(), 0700)
+		cobra.CheckErr(err)
+
+		// Make sure we're not overwriting an existing wallet (this should not happen)
+		walletFn := common.WalletFile()
+		f, _ := os.Open(walletFn)
+		if f != nil {
+			log.Fatalln("Wallet file already exists", walletFn)
+		}
```

Comment:
I like to use `os.Stat` for this:

```suggestion
		_, err := os.Stat(walletFn)
		switch {
			case errors.Is(err, fs.ErrNotExists):
				// all fine
			case err == nil:
				log.Fatalln("Wallet file already exists)
			default:
				log.Fatalln("Error opening %s: %v\n", walletFn, err)
		}
```